paddle/cinn/operator_fusion/graph_transformer/search_algorithm.h

```diff
@@ -138,8 +138,8 @@ struct SearchAlgorithm<ReverseTopoNodePairPattern,
 template <typename Kind, typename GraphMatcher, typename GraphOperation>
 void GraphTransformer(PatternGraph* graph) {
   VLOG(4) << "Start GraphTransformer...";
-  auto alog = SearchAlgorithm<Kind, GraphMatcher, GraphOperation>(graph);
-  alog();
+  auto along = SearchAlgorithm<Kind, GraphMatcher, GraphOperation>(graph);
```

Comment:
```suggestion
  auto algo = SearchAlgorithm<Kind, GraphMatcher, GraphOperation>(graph);
```

自己看上下文，这不是 `SearchAlgorithm` 返回一个 `Algorithm` 吗？